.github/workflows/release.yml

```diff
@@ -77,6 +64,20 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Ensure tagged commit is on main
+        run: |
+          echo "Tag: ${GITHUB_REF##*/}"
+          git fetch origin main
+          git merge-base --is-ancestor ${GITHUB_REF##*/} origin/main && echo "${GITHUB_REF##*/} is a commit on main!"
+
+      - name: Set version in package.json
+        run: cat ./package.json | jq ".version = \"$(echo -n ${GITHUB_REF##*/} | sed 's/^v\(.*\)/\1/' | tr -d '\n')\"" | tee ./package.json
```

Comment:
may be able to simplify this down to something closer to:
```bash
sed -i "s/\"version\": \"0.0.0\"/\"version\": \"${GITHUB_REF//v}\"/" package.json
```

Reply:
I considered that, but I ended up liking the JSON structural/syntactical guarantees of `jq` a bit more